MAUI/DataGrid/sorting.md

```diff
@@ -245,6 +245,34 @@ public class CustomSortComparer : IComparer<object>, ISortDirection
 {% endhighlight %}
 {% endtabs %}
 
+## Change sort icon color
+
+The default sort icon color can be customized by setting the `DataGridStyle.SortIconColor` property.
+{% tabs %}
+
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid"
+                        ItemsSource="{Binding OrderInfoCollection}" >
+        <syncfusion:SfDataGrid.DefaultStyle>
+                <syncfusion:DataGridStyle SortIconColor="DodgerBlue" />
+        </syncfusion:SfDataGrid.DefaultStyle>
+            <syncfusion:SfDataGrid.Columns>

```

Comment:
Columns tag should start with DefaultStyle tag's start position.

Reply:
changed